Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -311,8 +318,9 @@ void Update()
                 }
                 else
                 {
-                    // Player must click to attack
-                    if (InputManager.Instance.ActionStarted(InputManager.Actions.SwingWeapon))
+                    // Player must click or hold to attack, depending on attack mode
+                    if (InputManager.Instance.ActionStarted(InputManager.Actions.SwingWeapon)
+                        || (DaggerfallUnity.Settings.AttackMode == 2 && InputManager.Instance.IsSwingWeaponHeld()))
```

Comment:
You can replace this 'if' statement line with

```cs
 if (InputManager.Instance.ActionStarted(InputManager.Actions.SwingWeapon)
     || (InputManager.Instance.HasAction(InputManager.Actions.SwingWeapon)
        && DaggerfallUnity.Settings.AttackMode == 2))
```

And remove all changes from InputManager. What will happen is if the ActionStarted, then it will do ClickToAttack normally. When it hits the next frame, ActionStarted will be false, but it will still notice that SwingWeapon is still being held, and it will verify that the AttackMode is 2 (held) in order to proceed the swinging.